System/Constants.h

```diff
@@ -3,11 +3,31 @@
 
 namespace RTE {
 
+#pragma region Type Definitions
+	typedef int MOID; //!< Distinctive type definition for MovableObject IDs.
+	typedef int MID; //!< Distinctive type definition for Material IDs.
+#pragma endregion
+
 #pragma region Graphics Constants
-	static constexpr unsigned int c_PaletteEntriesNumber = 256; //!< Number of indexes in the graphics palette.
-	static constexpr unsigned int c_MOIDLayerBitDepth = 16; //!< Bit depth of MOID layer bitmap.
+	static constexpr unsigned short c_MaxScreenCount = 4; //!< Maximum number of player screens.
+	static constexpr unsigned short c_PaletteEntriesNumber = 256; //!< Number of indexes in the graphics palette.
+	static constexpr unsigned short c_MOIDLayerBitDepth = 16; //!< Bit depth of MOID layer bitmap.
+	static constexpr unsigned short c_GoldMaterialID = 2; //!< Index of gold material in the material palette.
+
+	enum ColorKeys {
+		g_KeyColor = 0,
+		//g_KeyColorS = 0, //!< Key color to cleat 8-bit bitmaps used for MOID silhouettes layers.
```

Comment:
Kill this commented junk! Also, 0xF81F is magenta fyi, maybe adjust/remove the comment.

Reply:
This doesn't bother me and I might possibly need it in the future.
Also 0xF81F (0,248,31) is not magenta fyi, it's the annoying bright green equivalent of magenta, but not pure green.
16711935 *is* magenta tho.

Reply:
This is a 16 bit colour thing though right? 0xF81F is not the same as 0x00F81F I think, plus we don't use that weird lime green anywhere do we? Maybe I'm wrong, I dunno, this colour/bpp business isn't my forte.

Assuming I'm not wrong though, I'd rather we rename it to TransparencyColor or something, so we know what the hell it is, and update the comment to mention that it's magenta. If you wanna look deeper and figure out all the places it's used so you can understand all its uses and update the name/comment accordingly, be my guest, but that's probably a lot of work haha.

Reply:
Well, *we* know what the hell it is. Calling it transparency is kind of misleading.
I mean, color key basically means chroma key.
The most correct term I can call this is a color mask.
Best I can do is `MaskColor`, because that's what it is.